dspot-maven/pom.xml

```diff
@@ -38,6 +39,14 @@
 					</execution>
 				</executions>
 			</plugin>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>2.12.4</version>
+				<configuration>
+					<skipTests>true</skipTests>
```

Comment:
Why are you skipping the test?

Reply:
I forgot to remove the DSpotMojoTest and remove the skip tests configuration.
I created DSpotMojoTest test only for debug issue to check the execution of dspot on test-projects.